apps/src/templates/census/CensusForm.jsx

```diff
@@ -31,159 +29,141 @@ export const censusFormPrefillDataShape = PropTypes.shape({
   schoolZip: PropTypes.string,
 });
 
-class CensusForm extends Component {
-  static propTypes = {
-    prefillData: censusFormPrefillDataShape,
-    initialSchoolYear: PropTypes.number,
-    schoolDropdownOption: PropTypes.object,
-    onSchoolDropdownChange: PropTypes.func,
+const CensusForm = ({
+  onSchoolDropdownChange,
+  schoolDropdownOption,
+  initialSchoolYear,
+  prefillData = {},
+}) => {
+  const [state, setState] = useState({
+    showFollowUp: false,
+    showPledge: false,
+    selectedHowMuchCS: [],
+    selectedTopics: [],
+    otherTopicsDesc: '',
+    schoolName: prefillData.schoolName ?? '',
+    schoolYear: initialSchoolYear,
+  });
+  const [submission, setSubmission] = useState({
+    name: prefillData.userName ?? '',
+    email: prefillData.userEmail ?? '',
+    role: prefillData.isTeacher ? 'TEACHER' : '',
+    country: prefillData.schoolCountry ?? 'United States',
+    hoc: '',
+    schoolName: prefillData.schoolName ?? '',
+    schoolCity: '',
+    schoolState: prefillData.schoolState ?? '',
+    schoolZip: prefillData.schoolZip ?? '',
+    schoolType: prefillData.schoolType ?? '',
+    afterSchool: '',
+    tenHours: '',
+    twentyHours: '',
+    otherCS: false,
+    followUpFrequency: '',
+    followUpMore: '',
+    acceptedPledge: false,
+    share: '',
+    optIn: '',
+  });
```

Comment:
for sure, that's a good call! especially since we never update two values at the same time, like a reset. but, yeah, my intention was to keep this as close to the original implementation as possible, particularly since there are no unit tests to use as a guide 😬 

I broke out the nested state into their own useStates to simplify things a bit hopefully!